couchbase/azuredeploy.json

```diff
@@ -0,0 +1,78 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "nodeCount": {
+      "type": "int",
+      "metadata": {
+        "description": "Number of nodes in the cluster"
+      }
+    },
+    "vmSize": {
+      "type": "string",
+      "metadata": {
+        "description": "Size of nodes in the cluster"
+      }
+    },
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "Admin username"
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Admin username"
+      }
+    }
+  },
+  "variables": {
+    "templateUrl": "https://raw.githubusercontent.com/benofben/azure-quickstart-templates/master/couchbase/nested/",
```

Comment:
should point at Azure/... instead of benofben/... and should follow #2 in the best practices checklist: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md